### DISCUSSION TITLE
Limit response with —json option with cli

### DISCUSSION THREAD

**Discussion by gituserjava** [OpenAPITools/openapi-diff#449] [Q&A]
Is there a way we can limit the response that is generated when “—json” option is used in command line?

For a huge api spec, this option is generating huge file with “oldApiSpec”, “newApiSpec”, “newEndpoints”, “oldEndpoints” and in my case for 2000 lines api spec, it created 2,26,822 lines json file. 

Can we just generate “newEndpoints”, “oldEndPoints” etc. I mean is there a way it can create json with the contents of text output?

**Comment by joschi** [comment]
> Is there a way we can limit the response that is generated when “—json” option is used in command line?

No, this is currently not possible.

You could post-process the JSON output with [`jq`](https://stedolan.github.io/jq/) or similar tools.
